setup_cfg_fmt.py

```diff
@@ -105,10 +104,14 @@ def cb(match: Match[str]) -> str:
 def _first_file(
     setup_cfg: str,
     prefix: str,
-    sort_key: Callable[[str], Any] | None = None,
 ) -> str | None:
     prefix = _case_insensitive_glob(prefix)
     path = _adjacent_filename(setup_cfg, prefix)
+
+    # make asciidoc last
+    def sort_key(filename: str) -> Any:
```

Comment:
this type is wrong

Reply:
yup youre right it wont change at any time so i can type it :ok_hand: